projects/ci-cd.md

```diff
@@ -0,0 +1,105 @@
+| GitHub         | [OpenTelemetry Actions Exporter - New Relic](https://github.com/marketplace/actions/new-relic-opentelemetry-github-actions-exporter)                        | [GitHub Action Repo](https://github.com/newrelic-experimental/gha-new-relic-exporter)              | * Metrics on workflow runs                           |
+| GitHub         | [Run with Telemetry GitHub Action](https://github.com/krzko/run-with-telemetry)                                                                             |                                                                                                    | * Traces via GitHub action                           |
+| GitLab         | [Distributed Tracing](https://docs.gitlab.com/ee/operations/tracing.html)                                                                                   |                                                                                                    | * Tracing in GitLab pipelines                        |
+| pytest         | [pyTest OpenTelemetry plugin](https://pypi.org/project/pytest-otel/)                                                                                        |                                                                                                    | * Tracing of Python test                             |
+| Maven          | [Maven OpenTelemetry extension](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/maven-extension/README.md)                           |                                                                                                    | * Tracing of Maven builds                            |
+| Ansible        | [Ansible OpenTelemetry Callback Plugin](https://docs.ansible.com/ansible/latest/collections/community/general/opentelemetry_callback.html)                  |                                                                                                    | * Tracing in Ansible playbooks                       |
+| JUnit          | [JUnit to OpenTelemetry](https://github.com/mdelapenya/junit2otlp)                                                                                          |                                                                                                    | * Tracing of JUnit test results                      |
+
+
+## Staffing / Help Wanted
+
+The goal of this project will consist of three stages:
+
+1. Working group preparation
+2. Specification
+3. Implementation
+
+### Required staffing
+
+All stages will require a project lead, domain experts, and engineers at a minimum. 
+
+* @horovits Project Lead (Logz.io)
+* @adrielp (Liatrio)
+* @thisthat (Dynatrace, Keptn)
+* @acramsay (Liatrio)
+* @krzko (ANZ Bank)
+* @Elfo404 (Grafana Labs)
+* @dsotirakis (Grafana Labs)
+* @xibz (Apple)
+* @e-backmark-ericsson (Ericsson)
+* @dnsmichi (GitLab)
```

Comment:
Adding two more GitLab team members, who agreed on joining the WG in https://gitlab.com/gitlab-org/gitlab/-/issues/338943#note_1684181306 

Darren is the product manager for Runner Core/Fleet, Sacha is the product manager for Observability. 

```suggestion
* @dnsmichi (GitLab)
* Darren Eastman (GitLab) 
* @sguyon (GitLab) 
```

Reply:
Done